packages/react-core/src/components/Alert/examples/Alert.md

```diff
@@ -147,14 +298,13 @@ import { Alert } from '@patternfly/react-core';
 </React.Fragment>
 ```
 
-### Inline plain variations
+### Plain inline alert variations
 
-It is not recommended to use an inline plain alert with actionClose nor actionLinks.
+It is not recommended to use an inline plain alert with `actionClose` nor `actionLinks`.
```

Comment:
As far as this PR goes, I think we should add a quick statement as to why one shouldn't do this.

As a possible follow-up, I feel like this example and the previous one could be combined into one, with the warning of not using `actionClose` and `actionLinks` being added onto the description of the previous example. Mainly I'm not sure if just adding a description should really be considered a variation of an alert.